contrib/jackc/pgx.v5/pgx_tracer.go

```diff
@@ -175,6 +183,43 @@ func (t *pgxTracer) TraceConnectEnd(ctx context.Context, data pgx.TraceConnectEn
 	finishSpan(ctx, data.Err)
 }
 
+func (t *pgxTracer) TraceAcquireStart(ctx context.Context, pool *pgxpool.Pool, _ pgxpool.TraceAcquireStartData) context.Context {
+	if !t.cfg.traceAcquire {
+		return ctx
+	}
+	opts := t.spanOptions(pool.Config().ConnConfig, operationTypeAcquire, "")
+	_, ctx = tracer.StartSpanFromContext(ctx, "pgx.acquire", opts...)
+	return ctx
+}
+
+func (t *pgxTracer) TraceAcquireEnd(ctx context.Context, pool *pgxpool.Pool, data pgxpool.TraceAcquireEndData) {
+	if t.cfg.traceHold {
+		opts := t.spanOptions(pool.Config().ConnConfig, operationTypeHold, "")
+		_, holdCtx := tracer.StartSpanFromContext(ctx, "pgx.hold", opts...)
```

Comment:
Is this going to make the `pgx.hold` span a child of `pgx.acquire` if that's enabled? In my mind I was thinking about it as a sibling span, but I guess I don't feel too strongly about it.

Reply:
right right, let's finish the acquire span first